src/components/List/Content/CTA/DeleteRowsButton.tsx

```diff
@@ -0,0 +1,75 @@
+import { useState } from 'react';
+import { useHistory, useLocation } from 'react-router-dom';
+import { useStoreActions } from '../../../../store';
+import { Tooltip } from '@mui/material';
+import ConfirmDialog from '../../../shared/ConfirmDialog';
+import EntityService from '../../../../services/entity/EntityService';
+import DeleteIcon from '@mui/icons-material/Delete';
+import _ from '../../../../services/translations/translate';
+
+interface DeleteRowsButtonProps {
+  entityService: EntityService;
+  selectedValues: Array<string | number>;
+}
+
+const DeleteRowsButton = (props: DeleteRowsButtonProps): JSX.Element => {
+  const { entityService, selectedValues } = props;
+  const location = useLocation();
+  const [showDelete, setShowDelete] = useState<boolean>(false);
+  const handleHideDelete = (): void => {
+    setShowDelete(false);
+  };
+  const history = useHistory();
+  const apiDelete = useStoreActions((actions: any) => {
+    return actions.api.delete;
+  });
+
+  const selectedValuesIds =
+    selectedValues.length > 1
+      ? `${selectedValues[0]}?_rmAlso[]=${selectedValues
+          .slice(1)
+          .join('&_rmAlso[]=')}`
+      : `${selectedValues}`;
```

Comment:
I think this would be easier to read
```suggestion
  let selectedValuesIds = selectedValues.pop();
  
  if (selectedValues.length > 1) {
      selectedValuesIds += `?_rmAlso[]=${selectedValues.join('&_rmAlso[]=')}`
  }
```

Reply:
perfect